src/target/cortexm.c

```diff
@@ -1062,6 +1062,25 @@ static uint32_t cortexm_dwt_func(target_s *target, target_breakwatch_e type)
 	}
 }
 
+static uint32_t cortexm_dwt_func_v8m(target_s *target, target_breakwatch_e type, size_t len)
+{
+	uint32_t x = CORTEXM_DWT_FUNC_ACTION_DEBUG_EVENT_V8M;
```

Comment:
Please don't use single letter variable names - something like `result` or `func` would be better here. Thank you for using `uint32_t` though!